tests/components/dynalite/test_light.py

```diff
@@ -67,3 +73,23 @@ async def test_remove_config_entry(hass, mock_device):
     assert await hass.config_entries.async_remove(entry_id)
     await hass.async_block_till_done()
     assert not hass.states.get("light.name")
+
+
+async def test_light_restore_state(hass, mock_device):
+    """Test restore from cache."""
+    mock_restore_cache(
+        hass,
+        [State("light.name", "on", attributes={ATTR_BRIGHTNESS: 77})],
+    )
+    await create_entity_from_device(hass, mock_device)
+    mock_device.init_level.assert_called_once_with(77)
+
+
+async def test_light_restore_state_bad_cache(hass, mock_device):
```

Comment:
Improve the test to also check that the state of the light is as expected.

Reply:
thanks. added